django/contrib/auth/base_user.py

```diff
@@ -21,6 +23,10 @@ def normalize_email(cls, email):
         """
         Normalize the email address by lowercasing the domain part of it.
         """
+        msg = 'BaseUserManager.normalize_email() is deprecated in favor of '
+        msg += 'AbstractUser.normalize_email() and will be removed in Django 4.1.'
+        warnings.warn(msg, RemovedInDjango41Warning, stacklevel=2)
```

Comment:
Copy the style of other warnings in the code base

```suggestion
        warnings.warn(
            'BaseUserManager.normalize_email() is deprecated in favor of '
            'AbstractUser.normalize_email() and will be removed in Django 4.1.',
            RemovedInDjango41Warning,
            stacklevel=2,
        )
```